driver/src/orderbook/util.rs

```diff
@@ -0,0 +1,62 @@
+use crate::models::{AccountState, Order};
+use ethcontract::Address;
+
+/// Filters empty orders as and account balances for tokens for which there is
+/// not at least one sell order by a user
+pub fn filter_auction_data(
```

Comment:
One comment I just though of, We now how many "filtered" things going on in the orderbook:
- `FilteredOrderbookReader`
- `OnchainFilteredOrderbookReader`
- `filter_auction_data`

Maybe this method should be renamed just to avoid confusion. Something like `canonicalize_auction_data`? I'm not 100% sold on the name, maybe you can think of something better.